planning/costmap_generator/schema/costmap_generator.schema.json

```diff
@@ -0,0 +1,146 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Costmap Generator Node",
+  "type": "object",
+  "definitions": {
+    "costmap_generator": {
+      "type": "object",
+      "properties": {
+        "costmap_frame": {
+          "type": "string",
+          "default": "map",
+          "description": "Created costmap's coordinate."
+        },
+        "vehicle_frame": {
+          "type": "string",
+          "default": "base_link",
+          "description": "Vehicle's coordinate."
+        },
+        "map_frame": {
+          "type": "string",
+          "default": "map",
+          "description": "Map's coordinate."
+        },
+        "update_rate": {
```

Comment:
Do you have any insight into the bounds for those parameters? For example, I expect the `update_rate` only makes sense to be positive.

Reply:
`update_rate > 0` makes sense. I've added it.

By the way, we could only infer the bounds manually from the description and experience. Is this what we are supposed to do for every potential bound?

Reply:
Ideally that would be derived by looking at what the code does. For example if a positive parameter is used as a denominator in a division, then ">0" should be used as a bound.